test/config/playwright.global.setup.ts

```diff
@@ -13,6 +13,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import checkWebServer from './playwright.check.webserver';
+import copyBpmnDiagrams from './copy.bpmn.diagram';
+import { FullConfig } from '@playwright/test';
 
-// only collect on chromium for now
-module.exports = module.exports = require('../helpers/config/jest-playwright').computeConfigurationForDevServerUsage('chromium');
+//  globalSetup file must export a single function.
+const globalSetup = async (config: FullConfig): Promise<void> => {
+  if (!config.rootDir.endsWith('bundles')) {
```

Comment:
⚠️ here we should test if the configuration is running a webserver instead.
The diagram and the webserver check are only needed in this case.
Changing the condition will make the code easier to understand: currently, I have to remember/understand that bundle tests are not using a webserver